fromgplus/fromgplus.php

```diff
@@ -56,6 +58,9 @@ function fromgplus_addon_settings(&$a,&$s) {
 	$s .= '<label id="fromgplus-label" for="fromgplus-account">'.t('Google Account ID').' </label>';
 	$s .= '<input id="fromgplus-account" type="text" name="fromgplus-account" value="'.$account.'" />';
 	$s .= '</div><div class="clear"></div>';
+	$s .= '<label id="fromgplus-keywords-label" for="fromgplus-keywords">'.t('Add keywords to post').'</label>';
```

Comment:
Yes, the `-a` parameter works as well. But I think when you are working on an addon and happen to update the strings, then the 1st call is more easy to remember.